IncubatingIntegrationTest/SetTest.cs

```diff
@@ -257,6 +257,176 @@ public async Task SetAddBatchAsync_ElementsAreStringEnumerable_RefreshTtl()
         Assert.Contains(element, set);
     }
 
+    [Theory]
+    [InlineData(null, "my-set", new byte[] { 0x00 })]
+    [InlineData("cache", null, new byte[] { 0x00 })]
+    [InlineData("cache", "my-set", null)]
+    public async Task SetRemoveElementAsync_NullChecksByteArray_ThrowsException(string cacheName, string setName, byte[] element)
+    {
+        await Assert.ThrowsAsync<ArgumentNullException>(async () => await client.SetRemoveElementAsync(cacheName, setName, element));
+    }
+
+    [Fact]
+    public async Task SetRemoveElementAsync_ElementIsByteArray_HappyPath()
+    {
+        var setName = Utils.NewGuidString();
+        var element = Utils.NewGuidByteArray();
+
+        await client.SetAddAsync(cacheName, setName, element, false);
+
+        // Remove element that is not there -- no-op
+        await client.SetRemoveElementAsync(cacheName, setName, Utils.NewGuidByteArray());
+        var fetchResponse = await client.SetFetchAsync(cacheName, setName);
```

Comment:
Maybe add comment seems like code block is all part of no-op comment above vs another test case
```suggestion
        // Fetch the whole set and make sure response has element we expect 
        var fetchResponse = await client.SetFetchAsync(cacheName, setName);
```